token/services/db/sql/common/tokens.go

```diff
@@ -1080,6 +1117,55 @@ func (t *TokenTransaction) StoreToken(ctx context.Context, tr driver.TokenRecord
 	return nil
 }
 
+func (t *TokenTransaction) SetSpendable(txID string, index uint64, spendable bool) error {
```

Comment:
We can use the tokenID instead of txID and index maybe. And use TxNum instead of uint64